.github/workflows/run_update_raster_stats.yml

```diff
@@ -0,0 +1,39 @@
+name: Update exposure raster stats
+
+on:
+  repository_dispatch:
+    types:
+      - trigger_exposure_raster_stats
+  workflow_dispatch:
+
+jobs:
+  run-script:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v4
+      with:
+        ref: exposure-pipeline
+
+    - name: Set up Python
+      uses: actions/setup-python@v5
+      with:
+        python-version: '3.11.4'
+        cache: 'pip'
+
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        python -m pip install -e .
+        python -m pip install -r requirements.txt
+
+    - name: Run script
+      env:
+        DEV_BLOB_SAS: ${{ secrets.DEV_BLOB_SAS }}
+        AZURE_DB_PW_DEV: ${{ secrets.AZURE_DB_PW_DEV }}
+        AZURE_DB_PW_PROD: ${{ secrets.AZURE_DB_PW_PROD }}
+        AZURE_DB_UID: ${{ secrets.AZURE_DB_UID_DEV }}
```

Comment:
Maybe want to change the name of this in `secrets` to not reference `DEV` -- not essential though.

Reply:
Yeah in general I think having a better naming structure for our env vars would be good. Following up with Mike on including them as organization secrets as well, which would force us to be consistent.